src/components/RemediationTable.js

```diff
@@ -133,26 +143,39 @@ function RemediationTable (props) {
 
     return (
         <React.Fragment>
-            <TableToolbar results={ filtered.length }>
-                <Level>
-                    <LevelItem>
+            <TableToolbar
+                className='ins-c-remediations-table__actions'
+                results={ filtered.length }
+                selected={ selector.getSelectedIds(remediationIds).length }>
+                <ToolbarGroup>
+                    <ToolbarItem>
                         <SimpleTableFilter buttonTitle="" placeholder="Search Playbooks" { ...filter.props } />
-                    </LevelItem>
-                    <LevelItem>
-                        <Split gutter="md">
-                            <SplitItem>
-                                <DeleteRemediationsButton
-                                    isDisabled={ !selector.getSelectedIds(remediationIds).length }
-                                    remediations={ selector.getSelectedIds(remediationIds) }
-                                />
-                            </SplitItem>
-                        </Split>
-                    </LevelItem>
-                </Level>
+                    </ToolbarItem>
+                </ToolbarGroup>
+                <ToolbarGroup>
+                    <ToolbarItem><Button> Create Remediation </Button></ToolbarItem>
+                    <ToolbarItem>
+                        <Button
+                            variant='link'
+                            isDisabled={ !selector.getSelectedIds(remediationIds).length }
+                            // If a user has a popup blocker, they may only get the last one selected
```

Comment:
@ryelo I think this^^ is going to affect a lot of people. If, instead of opening the tabs in parallel, we did this sequentially by waiting for the user to save each file this would not be an issue, right?